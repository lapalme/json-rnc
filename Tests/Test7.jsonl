{"a1":34}
{"a2":"bonjour"}
{"b":43}
{"b":"salut"}
