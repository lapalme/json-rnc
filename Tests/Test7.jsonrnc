start = a|b
a = {"a1":integer,"a2":string}
b = {"b":is}
is=integer|string