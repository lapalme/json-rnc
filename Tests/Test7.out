1:{'a1': 34}
{'a1': 34} does not match any alternative:
 -(#/definitions/a)	missing required field:a2	
 -(#/definitions/b)	missing required field:b	
(#/definitions/b)	unexpected field in object:a1	
2:{'a2': 'bonjour'}
{'a2': 'bonjour'} does not match any alternative:
 -	missing required field:a1	
 -	missing required field:b	
	unexpected field in object:a2	
4 objects read: 2 invalid, 0 bad, 0 with duplicate fields
Error Statistics
              1	{'a1': 34} does not match any alternative:
              1	{'a2': 'bonjour'} does not match any alternative:
